packages/mobile/src/harmony-native/components/input/TextInput/TextInput.tsx

```diff
@@ -266,7 +266,7 @@ export const TextInput = forwardRef(
               >
                 {StartIcon ? (
                   <StartIcon
-                    size={isSmall ? 's' : 'm'}
+                    size={isSmall ? 'l' : 'm'} // TODO adjust large variant accordingly
```

Comment:
same thing here, just want to make sure this is the new standard vs a one-off, in which case we can figure out what to do